src/main/fbs/histogram-tile.fbs

```diff
@@ -0,0 +1,44 @@
+namespace io.opentraffic.datastore.flatbuffer;
+
+enum VehicleType : byte {
+  Auto = 0,
+}
+
+struct Entry {
+  // day (0=Sunday,...,6=Saturday) * 24 + hour of day.
+  // note: could be 25 to account for the single 25th hour when DST changes?
+  day_hour:ubyte;
+
+  // index into the next_segment_ids array in the Segment.
+  // note: imposes a limit of 256 segments leaving any one segment.
+  next_segment_idx:ubyte;
+
+  // bucket in 5mph intervals.
+  // note: do we set a max speed bucket, e.g: 120mph?
+  speed_bucket:ubyte;
```

Comment:
Thanks for spotting that! I had forgotten to change that, but fixed in 2939adf.

I had been using it as a duration bucket throughout the code, but forgot to change the name of the field in the FB definition.